docs/nodes-and-miners/signer.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Run a sBTC Signer Node - Testnet
+description: Set up and run a sBTC signernode
+sidebar_position: 10
+---
+
+sBTC signer nodes are responsible for managing the signing procedure to process sBTC peg-out requests.
+
+All signing parties must run an instance of a signer node.
+
+To run a signer node, you’ll need access to run `bitcoind` in addition to a Stacks node.
+
+There is a minimum threshold for how many signer nodes need to be running at any given time in order for BTC stacking rewards to be paid out.
+
+## Running a Bitcoin Full Node
+
+The first step is to get a Bitcoin full node up and running.
+
+This part of the setup is the same as running a Stacks miner, so you can refer to the existing section in the docs on [setting up a full testnet Bitcoin node](https://docs.stacks.co/docs/nodes-and-miners/miner-testnet#running-a-bitcoin-testnet-full-node).
+
+## Running a Stacks Node
+
+Next you’ll need to get up and running with a Stacks node.
+
+Download a [stacks blockchain binary](https://github.com/stacks-network/stacks-blockchain/releases/latest), or [build from source](https://github.com/stacksfoundation/miner-docs/blob/main/stacks-blockchain.md#build-and-install-stacks-blockchain-from-source)
+
+_There may be some extra requirements to building, [defined here](https://github.com/stacksfoundation/miner-docs/blob/main/prerequisites.md#install-required-packages)_.
+
+:::tip
+It is recommended to use a persistent location for the chainstate, in the steps below we're using `/stacks-blockchain`
+:::
+
+### Generate a keychain
+
+Now, a keychain needs to be generated. With this keychain, we'll generate a master private key. From this master private key, we will derive the public keys necessary perform signing operations on both the Bitcoin and Stacks sides.
+
+To create a keychain, the simplest way is to use the [stacks-cli](https://docs.hiro.so/references/stacks-cli) with the `make_keychain` command.
+
+```bash
+npx @stacks/cli make_keychain -t 2>/dev/null | jq -r
+```
```

Comment:
Let's add instructions using the `blockstack-cli`

Reply:
https://github.com/stacks-network/cli-blockstack has notes but it says its deprecated in favor of @stacks/cli

Reply:
https://github.com/stacks-network/stacks-blockchain/blob/master/src/blockstack_cli.rs